.github/workflows/on-tag.yml

```diff
@@ -118,45 +122,39 @@ jobs:
         run: sha256sum images/*
 
       - name: Load images locally
-        run: ls images/ | xargs -I %  docker load -i "images/%"
+        run: ls images/ | xargs  -I %  docker load -i "images/%"
```

Comment:
I've started adding double spaces as a visual hint for things that _belong together_.  It helps me scan through stuff faster, but I reckon it might be confusing to others.  Still quite undecided on that one 🤷‍♂